.github/workflows/cross-browser-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Cross-Browser Test Flow
+
+on:
+  push:
+    branches: ['**']
+
+env:
+  # make chromedriver detect installed Chrome version and download the corresponding driver
+  DETECT_CHROMEDRIVER_VERSION: true
+
+jobs:
+  cross-browser-test:
+    name: Cross-Browser Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      # install Chrome first, so the correct version of webdriver can be installed by chromedriver when setting up the repo
+      - name: install Chrome stable
+        run: |
+          sudo apt-get update
+          sudo apt-get install google-chrome-stable
+      - name: install Firefox stable
+        run: |
+          sudo apt-get update
+          sudo apt-get install firefox
+      - uses: actions/checkout@v2
+      - name: Set up Node (14)
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+      - name: Bump Node memory limit
+        run: echo "NODE_OPTIONS=--max_old_space_size=4096" >> $GITHUB_ENV
+      - name: Test setup and yarn install
+        run: |
+          cp config/ci.config.json config/project.json
+          yarn
+      - name: yarn build
+        run: yarn build
+      - name: Run Cross Browser tests
+        run: xvfb-run yarn test:cross-browser
```

Comment:
Does this command run Firefox and Chrome in parallel? If not, maybe it's worth making 2 separate jobs?